src/clarity-angular/layout/tabs/aria-service.ts

```diff
@@ -0,0 +1,7 @@
+import {Injectable} from "@angular/core";
+
+@Injectable()
+export class AriaService {
```

Comment:
Do we have plans to / should we have plans to move this to utils so it can be re-used with other components?